typedb/common/rpc/stub.py

```diff
@@ -20,58 +20,52 @@
 #
 
 from abc import ABC
-from typing import TypeVar, Callable, Iterator
-
+from typing import Callable, Iterator, TypeVar
 
 import typedb_protocol.common.session_pb2 as session_proto
 import typedb_protocol.common.transaction_pb2 as transaction_proto
 import typedb_protocol.core.core_database_pb2 as core_database_proto
 import typedb_protocol.core.core_service_pb2_grpc as core_service_proto
+from grpc import Channel
 
-from grpc import Channel, RpcError
-
-from typedb.common.exception import TypeDBClientException
 
 T = TypeVar('T')
 
 
-def resilient_call(function: Callable[[], T]) -> T:
```

Comment:
`resilient_call` is now made to be an abstract method of `TypeDBStub`. `CoreStub` and `ClusterServerStub` will implement them in a different way.